src/client/datascience/notebook/helpers/helpers.ts

```diff
@@ -560,7 +560,8 @@ function translateDisplayDataOutput(
         items.push(new NotebookCellOutputItem(convertJupyterOutputToBuffer(key, data[key]), key));
     }
 
-    return new NotebookCellOutput(sortOutputItemsBasedOnDisplayOrder(items), metadata);
+    const x = sortOutputItemsBasedOnDisplayOrder(items);
```

Comment:
I think @DonJayamanne just converted this the other way in main if I recall correctly. I don't think that we want this file changed.

Reply:
Yup I'd remove the variable `x`